### ISSUE TITLE
Implement ClimateHack.AI model

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/power_perceiver#58]
- [x] train on our dataset.
- [x] reduce LR
- [x] try SSIM and (SSIM + MSE)
- [x] #59
- [x] OSGB coords (normalise!)
- [x] Only compute loss for the central 32x32 image
- [x] When plotting satellite data, use the same "exposure" across all frames.
- [x] Can we tell `create_u_net` not to skip the coord conv channels?
- [x] topography (set nans to zero. Normalise. Try to reuse as much of the existing code as possible ) 
- [x] Provide a full hour of history at 128x128
- [x] Sun's angle & azimuth at t0 at the center (implemented as a np_batch_processor) (to help the model understand brightness)
- [x] train for like 200 epochs
- [ ] make it more likely to sample from the UK
- [x] Plot topography
- [ ] Plot animations
- [x] LR scheduling. (Illinois presentation say they used Flat Cosine LR Policy, which I guess I could implement using [`torch.optim.lr_scheduler.CosineAnnealingLR`](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.CosineAnnealingLR.html). [Their code](https://github.com/jmather625/climatehack/blob/main/best-submission/train_unet.py#L53) uses [`ReduceLROnPlateau`](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html) [with](https://github.com/jmather625/climatehack/blob/main/best-submission/train_unet.py#L67) [`fit_flat_cos`](https://docs.fast.ai/callback.schedule.html#Learner.fit_flat_cos))
- [x] [Ranger21 Optimizer](https://github.com/lessw2020/Ranger21) (although Illinois used Ranger) (nope. Loss spikes up to insane values! Let's try `Ranger`)
- [x] [Ranger optimizer](https://github.com/lessw2020/Ranger-Deep-Learning-Optimizer). Buggy! Trying:
- [x] [Ranger optimizer from pytorch-forecasting](https://pytorch-forecasting.readthedocs.io/en/stable/api/pytorch_forecasting.optim.Ranger.html)
- [x] Try with blur_final=False (looks good!)
- [x] Try with last_cross=False (hurts a bit)
- [x] Try with last_cross=True and bottle=True
- [ ] Include better examples in the validation plots (at the moment, none of the examples clearly show clouds moving)
- [ ] check the osgb coord conv are sane after normalisation 
- [x] Plot more timesteps
- [ ] train on other "moving" datasets?!  Moving MNIST? Synthetic clouds?
- [x] Use 128x128 images?
    - [ ] Is there a way to _train_ on 128x128 but run inference at 64x64?
    - [ ] If not, maybe we should create 128x128 HRV batches for v15?
